server/app/com/xsn/explorer/util/RetryableFuture.scala

```diff
@@ -53,7 +71,16 @@ object RetryableFuture {
     RetryableFuture[A](delays) _
   }
 
+  /**
+   * calculates the delay for the given retry number
+   * @param baseDelay the delay for the first retry
+   * @param retry the retry for which the delay is being calculated
+   * @param factor how much the delay will grow with each retry. for example:
+   *               2 -> double the delay with each retry
+   *               3 -> triples the delay with each retry
+   * @return
+   */
   private def getDelay(baseDelay: Long, retry: Int, factor: Int = 2): FiniteDuration = {
```

Comment:
Update this to `private[util] def getDelay(...)` and then, write tests for the function.